OrbitGl/TimeGraph.cpp

```diff
@@ -535,17 +535,13 @@ void TimeGraph::UpdatePrimitives(PickingMode picking_mode) {
   uint64_t min_tick = GetTickFromUs(min_time_us_);
   uint64_t max_tick = GetTickFromUs(max_time_us_);
 
-  SortTracks();
-
-  float current_y = -layout_.GetSchedulerTrackOffset();
-
-  for (auto& track : sorted_tracks_) {
-    track->SetY(current_y);
-    track->UpdatePrimitives(min_tick, max_tick, picking_mode);
-    current_y -= (track->GetHeight() + layout_.GetSpaceBetweenTracks());
+  if (GOrbitApp->IsCapturing() || sorted_tracks_.empty()) {
```

Comment:
This breaks tracks filtering which is done inside of SortTracks()

Reply:
Fixed with the latest commit, thanks for finding this!